### PR TITLE
task/Add edit config dialog

### PR DESCRIPTION
**Pull request by Noloxs**
This includes major refactoring of code

NB: This will result in breaking changes cased by fixing the typo in the settings.json variables

All the features should be tested as refactoring has reached nearly every code section

- **Extract helper methods from specific classes**
- **Fix issue with changing loadout and serial**
- **Refactor pyqt5.py in to a series of smaller files**
- **Add shared view utilities to own class in view**
- **Rename test file for view**
- **Fix typo 'Strat e gem' to 'Strat a gem'**
- **Add possibility of changing stratagem bindings**
- **Add settings to executor tab**
- **Refactor code based on linting results**

### CODE REVIEW

**Comment by Noloxs** [comment]
Resolves: #13

File: src/view/pyqt5/util.py
Comment (stixes) [inline src/view/pyqt5/util.py:1]: Mixing class or non-class er ikke så godt.

File: tests/test_view_utilities.py
Comment (stixes) [inline tests/test_view_utilities.py:1]: Settings tests er i en Class. Burde disse også være :)

**Review by stixes — COMMENTED** [review]
Decent!

File: src/view/pyqt5/util.py
Comment (Noloxs) [reply src/view/pyqt5/util.py:1]: Burde man så have en Util ish class som alle de metoder så er tilgængelig igennem?

**Comment by Noloxs** [comment]
Will create an abstraction between executors and settings so that:

1. View can request menu items from the executor if any to add to the toolbar
2. EditConfig dialog can request a list of settings to present if any to the executor settings tab

**Review by stixes — APPROVED** [review]
This works. Suggest we move settings abstraction to improvement after merge 😁

**Pull request merged**